src/components/ChecTag.vue

```diff
@@ -110,6 +110,14 @@ export default {
     .chec-tag__dismiss {
       @apply bg-white text-gray-600;
     }
+
+    &.chec-tag--disabled {
+      @apply bg-gray-600 text-white cursor-not-allowed border-none;
```

Comment:
```suggestion
      @apply bg-gray-600 text-white;
```

These styles have a specificity of 2, so lets keep the number of styles we're defining nice and low.